source/cgame/cg_events.cpp

```diff
@@ -1797,7 +1797,7 @@ static void CG_FirePlayerStateEvents( void )
 	            reactionKillsDelay += cg.realFrameTime * 1000;
 
 	            if( reactionKillsDelay > cg_reactionKillsTimeout->integer ){     
-                CG_SexedSound( cg.view.POVent, CHAN_ANNOUNCER, va( S_PLAYER_RKILL_1_to_2, ( 1+rand()%3 ) ),
+                CG_SexedSound( cg.view.POVent, CHAN_ANNOUNCER, va( S_PLAYER_RKILL_1_to_50, ( 1+rand()%50 ) ),
```

Comment:
shouldn't this be 51?

Reply:
the reaction kills used 3 voice lines, not 2 like it says so i guess whoever implemented them first didn't do it correctly 🤷